perses/tests/testsystems.py

```diff
@@ -307,7 +307,7 @@ def __init__(self, **kwargs):
         from pkg_resources import resource_filename
         valence_xml_filename = resource_filename('perses', 'data/amber99sbildn-valence-only.xml')
         system_generators['vacuum'] = SystemGenerator([valence_xml_filename],
-            forcefield_kwargs={ 'nonbondedMethod' : app.NoCutoff, 'implicitSolvent' : None, 'constraints' : None },
+            forcefield_kwargs={ 'nonbondedMethod' : app.NoCutoff, 'implicitSolvent' : None, 'constraints' : app.HBonds },
```

Comment:
Do you want to add `app.HBonds` constraints as default to the `SmallMoleculeLibraryTestSystem` base class? You could make it an option to the constructor so that you can have some test systems both be with and without constraints.